apisix/plugins/openid-connect.lua

```diff
@@ -71,6 +72,10 @@ local schema = {
             type = "string",
             description = "the URI will be redirect when request logout_path",
         },
+        symmetric_key = {
+            type = "string",
+            description = "base64url-encoded secret for verifying HS??? tokens",
```

Comment:
What's `HS???` token?

Reply:
like HS256, HS512

Reply:
we can just use `HS256, HS512`, not `HS???`,  it looks unclear

Reply:
ok. sure :)

Reply:
I updated